### ISSUE TITLE
Is there a way I can make this work with `mac-mwheel-scroll`?

### ISSUE THREAD

**Issue by livtanong** [ahungry/fast-scroll#4]

**Comment by ahungry** [comment]
Sure, what's the actual command that executes on that event?  is it actually `mac-mwheel-scroll` when you press `C-h k` and then scroll the wheel?

If so, I can add an advice wrapper for that, similar to the other advices I have in here.

**Comment by livtanong** [comment]
Yeah, `mac-mwheel-scroll` is the event. I think the scroll direction is also inside the event passed to `mac-mwheel-scroll`.

**Comment by ahungry** [comment]
Give this a try in your config file (assuming you've already included/set up this package):

```elisp
(advice-add #'mac-mwheel-scroll :around #'fast-scroll-run-fn-minimally)
```

But please note the timer to cause this to kick in is very low at the moment (`0.05` seconds or so), so it will tend to activate during auto keypress scrolling events from a machine, but i'm not sure if scroll wheel event would be fast enough (you can tell if it's working by the disappearance of font-lock colors/faces when it activates)

**Comment by livtanong** [comment]
Just tried it, and it does flicker during scroll. Also, funny thing: You know how on mac, you can scroll a window that isn't focused? If I have a frame split into two windows, and focus on window 1, and scroll on window 2, it's window 1 that flickers the font lock. 😆 

I suppose fast-scroll isn't really designed for use with mouse wheels.

**Comment by ahungry** [comment]
hah, not tested extensively (much or at all) for them.  I would be happy to welcome any additions to the code that improved it in that area though (I don't use a mouse wheel myself)

**Comment by livtanong** [comment]
Unfortunately I'm not the right person for that job. :P Emacs newbie here
